### DISCUSSION TITLE
Non-Public Drive Photos accessible via the Slides API?

### DISCUSSION THREAD

**Discussion by jackwilmerding** [googleapis/googleapis#866] [Ideas]
Hey folks, posted this a while back on the python api client repo without realizing that that repo was for the python client only. Any shot someone here could let me know if this is even possible? It'd really help a lot of my slides workflows. Thanks!

"Hi. I'm using a client oauth that has both full Drive and full Slides scope. Given this, why can I not use private drive images with the CreateImageRequest? This is somewhat experience-breaking, as most people do not want to make personal/school/work images for slide decks public.

I was following [this tutorial](https://www.youtube.com/watch?v=8LSUbKZq4ZY) on youtube.

Here was the exact error message: <HttpError 400 when requesting [request url] returned "Invalid requests[0].createImage: There was a problem retrieving the image. The provided image should be publicly accessible, within size limit, and in supported formats.". Details: "Invalid requests[0].createImage: There was a problem retrieving the image. The provided image should be publicly accessible, within size limit, and in supported formats.">

I have confirmed these images exist at the provided URIs and comply with image size and format restrictions, so it's definitely a private/public image issue. If a google developers project has credentials for both services, is it not possible to get these private images? Thanks for considering my question."

**Comment by zevburton** [comment]
Would be seriously helpful, especially within business contexts!

**Reply by msimpson-CL** [reply]
Yes, we'd love to use the google APIs but simply cannot make our business info publicly available.

**Comment by mayurankv** [comment]
I'm also heavily interested in this!
